core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectory.java

```diff
@@ -249,6 +250,8 @@ public String toString() {
    * @return the {@link InodeDirectory} representation
    */
   public static InodeDirectory fromJournalEntry(InodeDirectoryEntry entry) {
+    // If journal entry has no security enabled, set default mode for backwards-compatibility.
```

Comment:
nit: update comment to say "If the journal entry has no mode set ..." same for `InodeFile`